docs/wiki/EL/el-clients.md

```diff
@@ -1,60 +1,98 @@
+[Why?](https://clientdiversity.org/#why)
+
 
 ### Besu
+Developed by the Hyperledger Foundation in Java, Besu (Hyperledger Besu) is distinguished for its enterprise-grade features and compatibility with various Hyperledger projects.
+It supports both public and private networks such as Rinkeby, Ropsten, and Goerli, offering robust command-line tools and a JSON-RPC API.
 
-Developed by Hyperledger Foundation in Java
+Noteworthy Features:
+- [Private Networks](https://besu.hyperledger.org/private-networks/)
+- [Pruning](https://besu.hyperledger.org/public-networks/how-to/bonsai-limit-trie-logs#prune-command-for-mainnet)
+- [Monitoring](https://besu.hyperledger.org/private-networks/how-to/monitor)
+- [Parallel Transaction Execution](https://besu.hyperledger.org/public-networks/concepts/parallel-transaction-execution)
 
 ### Erigon
+Initially a fork of Geth (Go Ethereum), Erigon focuses on optimizing performance, fast synchronization capabilities, and reducing disk space usage.
+Erigon's architecture allows it to complete a full archive node sync in under three days with less than 2 TB of data storage, making it ideal for resource-efficient node deployments.
 
-Originally fork of geth called turbo-geth
+Noteworthy Features:
+- [Supported Networks](https://erigon.gitbook.io/erigon/basic-usage/supported-networks)
+- [Pruning](https://erigon.gitbook.io/erigon/basic-usage/usage/type-of-node#full-node-or-pruned-node)
 
 ### Geth
+As the official Go implementation of Ethereum, Geth (Go Ethereum) enjoys widespread adoption among developers and users alike.
+It supports various node types (full, light, archive) and is renowned for its extensive toolset and community support.
+Geth's flexibility in deployment—through package managers, Docker containers, or manual setup—ensures its versatility in diverse blockchain environments.
 
-Go-ethereum, developed by a team in EF
+Noteworthy Features:
+- [Pruning](https://geth.ethereum.org/docs/fundamentals/pruning)
+- [Custom EVM Tracer](https://geth.ethereum.org/docs/developers/evm-tracing/custom-tracer)
+- [Monitoring Dashboards](https://geth.ethereum.org/docs/monitoring/dashboards)
 
 ### Nethermind
+Written in C# .NET, Nethermind is designed for stability and integration with existing tech infrastructures.
+It offers optimized virtual machine performance, comprehensive analytics support via Prometheus/Grafana dashboards, and robust security features. 
+Nethermind is suitable for both private Ethereum networks and decentralized application (dApp) development, emphasizing data integrity and scalability.
 
-Developed in .NET
-
-## Clients in development
+Noteworthy Features:
+- [Private Networks](https://docs.nethermind.io/fundamentals/private-networks)
+- [Prometheus and Grafana](https://docs.nethermind.io/monitoring/metrics/grafana-and-prometheus)
 
 ### Reth
+Reth (Rust Ethereum) is a modular and efficient Ethereum client designed for user-friendliness and high performance.
```

Comment:
here I would put emphasis on the performance and it's inspiration from erigon architecture